### PR TITLE
Create README FOR MPLS2-2

### PR DESCRIPTION
**Pull request by ihebboubaker**
Validate static LSP functionality with BGP resolved next-hop. This test verifies that the DUT can forward MPLS traffic based on a static LSP that uses a next-hop resolved via BGP.

This is to update this PR https://github.com/openconfig/featureprofiles/pull/3106#discussion_r1777251673 comments

### CODE REVIEW

**Comment by dplore** [comment]
should we abandon #3106 and just review this one?

**Review by dplore — DISMISSED** [review]

File: feature/mpls/otg_tests/static_bgp_nexthop/README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+*   Verify that traffic is discarded.
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+  ## Config paths
+  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/next-hop:
+  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/incoming-label:
+
```
Comment (dplore) [inline feature/mpls/otg_tests/static_bgp_nexthop/README.md:109]:
```suggestion
  ## Config paths
  /network-instances/network-instance/protocols/protocol/static-routes/config/prefix
  /network-instances/network-instance/protocols/protocol/static-routes/config/index:
  /network-instances/network-instance/protocols/protocol/static-routes/next-hops/next-hop/config/next-hop:

  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/name: 
  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/next-hop:
  /network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/incoming-label:
```

Comment (ihebboubaker) [reply feature/mpls/otg_tests/static_bgp_nexthop/README.md]: Hey Daren,

Those paths are not invalid OCPaths

Comment (dplore) [reply feature/mpls/otg_tests/static_bgp_nexthop/README.md]: Sorry about that!  Note,  in future you can find and correct these yourself by searching at https://openconfig.net/projects/models/paths/

Here's my suggestion though:

```
/network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/incoming-label 
/network-instances/network-instance/mpls/lsps/static-lsps/static-lsp/egress/config/next-hop 

/network-instances/network-instance/protocols/protocol/static-routes/static/config/prefix 
/network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/next-hop
/network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/index
```

**Review by dplore — APPROVED** [review]

**Pull request merged**